ev3dev/core.py

```diff
@@ -2584,8 +2584,8 @@ def rate_and_angle(self):
 
         return self.value(0), self.value(1)
 
-class InfraredSensor(Sensor):
 
+class InfraredSensor(Sensor):
```

Comment:
I had to look at this for a minute to figure out what in the world this change was.  It is because I deleted the blank line between the `class InfraredSensor` and the `"""`.  It is odd that it displayed it that way though